linera-base/src/crypto.rs

```diff
@@ -172,6 +180,29 @@ impl Serialize for CryptoHash {
     }
 }
 
+impl Serialize for CryptoHashVec {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: serde::ser::Serializer,
+    {
+        serializer.serialize_newtype_struct("CryptoHashVec", &self.0)
```

Comment:
Another `#[transparent]`?

Reply:
Ah, good point.

Reply:
Done in https://github.com/linera-io/linera-protocol/pull/3101/commits/dca5da358b8cda100075937b57c5c9ac61169a4e.

Reply:
Actually, this won't work, b/c we will start getting the errors that we can call `type_name()` only on `struct` or `newtype` (b/c we'll call `Deserialize` on the `Vec<_>` and that will fail).

Reply:
Also why do we need this in the first place?